generate-config-schema.js

```diff
@@ -8,4 +8,5 @@ const configurationSchema = require("./node_modules/markdownlint/schema/markdown
 // Update package.json
 const configurationRoot = packageJson.contributes.configuration.properties["markdownlint.config"];
 configurationRoot.properties = configurationSchema.properties;
+configurationRoot.additionalProperties = configurationSchema.additionalProperties || false;
```

Comment:
Please just use the value - don't fall back to `false`.

Reply:
Updated in c086768.